clients/include/utility.hpp

```diff
@@ -148,6 +146,9 @@ static constexpr bool is_complex = false;
 #define HIPSOLVER_CLANG_STATIC
 #endif
 
+template <typename T>
+HIPSOLVER_CLANG_STATIC constexpr bool is_complex = false;
+
```

Comment:
What's the problem?

Reply:
Oh. I see. That's your workaround for the cppcheck syntax error.

I've been having trouble reproducing the scan results myself, too. Nothing here is urgent, so I'd like to ensure we've fully addressed the problem before moving forward. We probably just need to find the right person to ask about reproducing the error locally.

Reply:
Looks like the syntax error is still present in my local results. I guess we either have to suppress the warning or wait for an update to cppcheck?